src/gbo/GameBoyCore.js

```diff
@@ -4986,7 +4986,7 @@ GameBoyCore.prototype.recomputeDimension = function () {
 	//Cache some dimension info:
 	// this.onscreenWidth = this.canvas.width;
 	// this.onscreenHeight = this.canvas.height;
-	if (window && window.mozRequestAnimationFrame) {
+	if (window && window.mozRequestAnimationFrame || (navigator.userAgent.toLowerCase().indexOf("gecko") != -1 && navigator.userAgent.toLowerCase().indexOf("like gecko") == -1)) {
```

Comment:
This is a shitshow riiiiight here. Firefox gets canvas scaled by CSS and webkit/blink based browsers get canvas scaled by context2d. The reason being Firefox doesn't hw accelerate context2d on the platform I test on while webkit/blink does, while at the same time Firefox accelerates scaling via CSS when webkit/blink do not.

Newer Firefox removed the deprecated mozRequestAnimationFrame we were sniffing on, so it defaulted to the webkit/blink friendly path when it shouldn't in Firefox 4X.